apis/Google.LongRunning/Google.LongRunning/OperationsClient.cs

```diff
@@ -58,189 +58,41 @@ private OperationsSettings(OperationsSettings existing) : base(existing)
-        /// <item><description>Maximum delay: 60000 milliseconds</description></item>
-        /// <item><description>Delay multiplier: 1.3</description></item>
-        /// </list>
-        /// </remarks>
-        public static gaxgrpc::BackoffSettings GetDefaultRetryBackoff() => new gaxgrpc::BackoffSettings(
-            delay: sys::TimeSpan.FromMilliseconds(100),
-            maxDelay: sys::TimeSpan.FromMilliseconds(60000),
-            delayMultiplier: 1.3
-        );
-
-        /// <summary>
-        /// "Default" timeout backoff for <see cref="OperationsClient"/> RPC methods.
-        /// </summary>
-        /// <returns>
-        /// The "Default" timeout backoff for <see cref="OperationsClient"/> RPC methods.
-        /// </returns>
-        /// <remarks>
-        /// The "Default" timeout backoff for <see cref="OperationsClient"/> RPC methods is defined as:
-        /// <list type="bullet">
-        /// <item><description>Initial timeout: 90000 milliseconds</description></item>
-        /// <item><description>Timeout multiplier: 1.0</description></item>
-        /// <item><description>Maximum timeout: 90000 milliseconds</description></item>
-        /// </list>
-        /// </remarks>
-        public static gaxgrpc::BackoffSettings GetDefaultTimeoutBackoff() => new gaxgrpc::BackoffSettings(
-            delay: sys::TimeSpan.FromMilliseconds(90000),
-            maxDelay: sys::TimeSpan.FromMilliseconds(90000),
-            delayMultiplier: 1.0
-        );
-
         /// <summary>
         /// <see cref="gaxgrpc::CallSettings"/> for synchronous and asynchronous calls to
         /// <c>OperationsClient.GetOperation</c> and <c>OperationsClient.GetOperationAsync</c>.
         /// </summary>
         /// <remarks>
-        /// The default <c>OperationsClient.GetOperation</c> and
-        /// <c>OperationsClient.GetOperationAsync</c> <see cref="gaxgrpc::RetrySettings"/> are:
-        /// <list type="bullet">
-        /// <item><description>Initial retry delay: 100 milliseconds</description></item>
-        /// <item><description>Retry delay multiplier: 1.3</description></item>
-        /// <item><description>Retry maximum delay: 60000 milliseconds</description></item>
-        /// <item><description>Initial timeout: 90000 milliseconds</description></item>
-        /// <item><description>Timeout multiplier: 1.0</description></item>
-        /// <item><description>Timeout maximum delay: 90000 milliseconds</description></item>
-        /// </list>
-        /// Retry will be attempted on the following response status codes:
-        /// <list>
-        /// <item><description><see cref="grpccore::StatusCode.DeadlineExceeded"/></description></item>
-        /// <item><description><see cref="grpccore::StatusCode.Unavailable"/></description></item>
-        /// </list>
-        /// Default RPC expiration is 600000 milliseconds.
+        /// FIXME: Generate with retries.
```

Comment:
These are to be fixed before GA?
Just wanted to check that they're not something you wanted to fix in this PR?

Reply:
Nope, not for this PR. We basically need google.longrunning to get some annotations, then we can move it to the microgenerator. There's a method that we either don't want to generate or want to put a caveat around, but that's a different matter. I'll see if we can get the annotations in this week, then another PR will move it to the microgenerator and fix these.